Sources/StreamChat/APIClient/Endpoints/Payloads/PollsPayloads.swift

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct PollPayload: Decodable {
+    let duration: String
+    let poll: PollResponseData
+
+    init(duration: String, poll: PollResponseData) {
+        self.duration = duration
+        self.poll = poll
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case duration
+        case poll
+    }
+}
+
+struct QueryPollsPayload: Decodable {
+    let duration: String
+    let polls: [PollResponseData]
+    var next: String?
+    var prev: String?
+
+    init(duration: String, polls: [PollResponseData], next: String? = nil, prev: String? = nil) {
+        self.duration = duration
+        self.polls = polls
+        self.next = next
+        self.prev = prev
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case duration
+        case polls
+        case next
+        case prev
+    }
+}
+
+struct PollResponseData: Decodable {
+    let allowAnswers: Bool
+    let allowUserSuggestedOptions: Bool
+    let answersCount: Int
+    let createdAt: Date
+    let createdById: String
+    let description: String
+    let enforceUniqueVote: Bool
+    let id: String
+    let name: String
+    let updatedAt: Date
+    let voteCount: Int
+    let votingVisibility: String
+    let options: [PollOptionResponseData?]
+    let ownVotes: [PollVoteResponseData?]
```

Comment:
We don't have this "ResponseData" postfix anywhere in the codebase, to make things consistent, all the payloads should have a `Payload` post-fix, ideally 
```suggestion
    let options: [PollOptionResponseData?]
    let ownVotes: [PollVoteResponseData?]
```